Props/umbrellas

MrKoddy opened the pull request:
#362

MrKoddy committed:
Colors for umbrellas

Add tint palette for more colors

MrKoddy committed:
Colors for umbrellas

Rigonkmalk commented on resources/[props]/soz-prop-umbrella-colors/p_amb_brolly_01.ydr.xml:
Why your files are placed here, and you have a duplicate file in `resources/[props]/soz-prop-umbrella-colors/p_amb_brolly_01/p_amb_brolly_01.ydr.xml` also ?

Rigonkmalk commented on resources/[props]/soz-prop-umbrella-colors/p_amb_brolly_01/lev_des.ytyp.xml:
Do not modify Vanilla files, if you try to add or modify contents or properties of a vanilla files, please add a custom name on the prefix and create a fully new ytyp files.

Rigonkmalk commented on resources/[props]/soz-prop-umbrella-colors/p_amb_brolly_01/mrk_prop_umbrella_tint_pal.dds:
DDS files are not loaded by FiveM, please add them in a YTD files.

Rigonkmalk commented on resources/[props]/soz-prop-umbrella-colors/p_amb_brolly_01/p_amb_brolly_01.ydr.xml:
Duplicated files ?

Rigonkmalk commented on resources/[props]/soz-prop-umbrella-colors/p_amb_brolly_01/p_amb_brolly_01/mrk_prop_umbrella_tint_pal.dds:
DDS files are not loaded by FiveM, please add them in a YTD files.

Following all of your DDS files inside this folder.

Rigonkmalk reviewed (commented):
Hello @MrKoddy and thank you for your PR.

Please follow all of my review and fix them before next steps.

Laikker commented:
I don't see any code, how would it be possible to choose the color?

MrKoddy commented:
I'll make changes you mention here. I didn't know I should write some code and I can't code. I made this tint palette following a tutorial which didn't talk about coding so player can chose the color in fivem. I still can transform the PR into indivdual files so we have many colors for umbrellas

MrKoddy committed:
Umbrellas variants

Sorry for the delay, I was on another project. I edited the props and followed the instructions, everything should be ok. I didn't do with tint palette because I can't program. So there 8 umbrellas variants

MrKoddy commented:
I pushed a commit. I think everything is good. Making the texture dictionary didn't work for this project

MrKoddy committed:
Umbrella and ytyp update

Updated a color and ytyp

MrKoddy committed:
remove old files

Laikker commented:
Hello,
I have integrated this in the main repo.
Thanks
